modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/SqlDdlParserTest.java

```diff
@@ -767,15 +770,36 @@ public void dropIndexIfExists() {
     }
 
     /**
-     * Ensures that the user cannot use the TIME_WITH_LOCAL_TIME_ZONE and TIMESTAMP_WITH_LOCAL_TIME_ZONE types for table columns.
+     * Ensures that the parser throws the expected exception when attempting to use the following unsupported types for table columns.
+     *
+     * <ol>
+     *     <li>{@link SqlTypeName#TIME_TZ TIME WITH TIME ZONE}</li>
+     *     <li>{@link SqlTypeName#TIME_WITH_LOCAL_TIME_ZONE TIME WITH LOCAL TIME ZONE}</li>
+     *     <li>{@link SqlTypeName#TIMESTAMP_TZ TIMESTAMP WITH TIME ZONE}</li>
+     * </ol>
      */
     // TODO: Remove after https://issues.apache.org/jira/browse/IGNITE-21555 is implemented.
-    @Test
-    public void timeWithLocalTimeZoneIsNotSupported() {
+    @ParameterizedTest(name = "ензу={0}")
+    @CsvSource({
+            "TIME WITH TIME ZONE, Encountered \"WITH\"",
+            "TIME WITH LOCAL TIME ZONE, Encountered \"WITH\"",
+            "TIME WITHOUT TIME ZONE, Encountered \"WITHOUT\"",
+            "TIME WITHOUT LOCAL TIME ZONE, Encountered \"WITHOUT\"",
+            "TIMESTAMP WITH TIME ZONE, Encountered \"TIME\"",
+            "TIMESTAMP WITHOUT TIME ZONE, Encountered \"WITHOUT\"",
```

Comment:
do we need to treat "TIMESTAMP WITHOUT TIME ZONE" as simple TIMEZONE syntax ?

Reply:
Added `WITHOUT TIME ZONE` syntax.